pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -669,6 +673,17 @@ def test_get_indexer_pad_requires_monotonicity(self):
         with pytest.raises(ValueError, match=msg):
             rng2.get_indexer(rng, method="pad")
 
+    @pytest.mark.parametrize("box", [IntervalIndex, array, list])
+    def test_get_indexer_interval_index(self, box):
```

Comment:
Could you move this to `pandas/tests/indexes/interval/test_indexing.py`

Reply:
Sure. Moved